core/src/test/scala/org/apache/spark/status/AppStatusStoreSuite.scala

```diff
@@ -227,32 +268,41 @@ class AppStatusStoreSuite extends SparkFunSuite {
     liveTask.write(store.asInstanceOf[ElementTrackingStore], 1L)
   }
 
-  private def getTaskMetrics(i: Int): TaskMetrics = {
+  /**
+   * Creates fake task metrics
+   * @param seed The random seed. The output will be reproducible for a given seed.
+   * @return The test metrics object with fake data
+   */
```

Comment:
The previous version of the test set every metric to the same value. This left a gap in test coverage where the test would pass if the code accidentally used the wrong metric (an easy mistake to make). This improves the test coverage be eliminating this gap because every metric has unique values. This is important because I expanded the test to check every metric, instead of just `executorRunTime` like before.

Reply:
Sounds good.